AENS.md

```diff
@@ -125,7 +125,36 @@ This can have multiple entries, e.g. a payment address associated
 with the name and an oracle address associated with the name.
 
 
-### Name
+A subname entry (or, subdomain), belongs to a name entry.
```

Comment:
This document shall be clarified on what is for Lima release onwards.

Reply:
added